src/components/ComponentPages/CreateNewCamp/CampUI.tsx

```diff
@@ -52,203 +52,201 @@ const CreateNewCampUI = ({
-                        name="nick_name"
-                        {...nickNmRule}
-                        initialValue={nickNameList[0]?.id}
-                      >
-                        <Select
-                          placeholder={placeholders.nickName}
-                          allowClear
-                          size={"large"}
-                        >
-                          {nickNameList.map((nick) => (
-                            <Option key={nick.id} value={nick.id}>
-                              {nick.nick_name}
-                            </Option>
-                          ))}
-                        </Select>
-                      </Form.Item>
-                    ) : null}
-                    {nickNameList.length <= 0 ? (
-                      <Form.Item
-                        label={labels.cr_nick_name}
-                        name="nick_name"
-                        {...nickNmRule}
-                      >
-                        <Select
-                          placeholder={placeholders.nickName}
-                          allowClear
-                          size={"large"}
-                        ></Select>
-                      </Form.Item>
-                    ) : null}
-                  </Col>
-                  <Col xs={24} sm={12}>
-                    {parentCamp.length > 0 ? (
-                      <Form.Item
-                        label={labels.cr_parent_camp}
-                        name="parent_camp_num"
-                        {...parentCampRule}
-                        initialValue={
-                          crCamp.camp_num || topicData?.parent_camp_num
-                        }
-                      >
-                        <Select
-                          allowClear
-                          size={"large"}
-                          placeholder="Parent camp"
-                        >
-                          {parentCamp.map((camp) => (
-                            <Option value={camp.camp_num} key={camp.id}>
-                              {camp.camp_name}
-                            </Option>
-                          ))}
-                        </Select>
-                      </Form.Item>
-                    ) : null}
-                    {parentCamp.length <= 0 ? (
-                      <Form.Item
-                        label={labels.cr_parent_camp}
-                        name="parent_camp_num"
-                        {...parentCampRule}
-                      >
-                        <Select
-                          allowClear
-                          size={"large"}
-                          placeholder="Parent camp"
-                        ></Select>
-                      </Form.Item>
-                    ) : null}
-                  </Col>
-                </Row>
-                <Row gutter={16}>
-                  <Col xs={24} sm={12}>
+          <Card title={CardTitle} className="can-card-style">
+            <Form
```

Comment:
Can we move this form in a separate component?

Reply:
Done.